### PR TITLE
Purview, protocol client

### PR DESCRIPTION
**Pull request by qwordy**

### CODE REVIEW

File: sdk/purview/azure-analytics-purview-account/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[samples]: src/samples/java/com/azure/analytics/purview/catalog
+[source_code]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/purview/azure-analytics-purview-catalog/src
+[samples_code]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/purview/azure-analytics-purview-catalog/src/samples/
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-account/README.md:89]: Fix your readme!

File: sdk/purview/azure-analytics-purview-account/src/samples/java/com/azure/analytics/purview/account/GetAccount.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.azure.analytics.purview.account;
+
+import com.azure.core.util.BinaryData;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public class GetAccount {
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-account/src/samples/java/com/azure/analytics/purview/account/GetAccount.java:6]: If possible have another test on pageable, e.g. "/collections" API.

File: sdk/purview/azure-analytics-purview-account/src/main/java/com/azure/analytics/purview/account/PurviewAccountClientBuilder.java [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+                (configuration == null) ? Configuration.getGlobalConfiguration() : configuration;
+        if (httpLogOptions == null) {
+            httpLogOptions = new HttpLogOptions();
+        }
+        List<HttpPipelinePolicy> policies = new ArrayList<>();
+        if (tokenCredential != null) {
+            policies.add(new BearerTokenAuthenticationPolicy(tokenCredential, DEFAULT_SCOPES));
+        }
+        String clientName = properties.getOrDefault(SDK_NAME, "UnknownName");
+        String clientVersion = properties.getOrDefault(SDK_VERSION, "UnknownVersion");
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-account/src/main/java/com/azure/analytics/purview/account/PurviewAccountClientBuilder.java:247]: Where does this `properties` get its value?

File: sdk/purview/azure-analytics-purview-account/README.md (hunk 1 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-account/README.md]: Fixed

File: sdk/purview/azure-analytics-purview-account/src/samples/java/com/azure/analytics/purview/account/GetAccount.java (hunk 2 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-account/src/samples/java/com/azure/analytics/purview/account/GetAccount.java:6]: Added. But unauthorized error.

File: eng/versioning/version_client.txt [hunk 4]
```diff
@@ -44,8 +44,9 @@ com.azure:azure-ai-metricsadvisor;1.0.0;1.1.0-beta.1
 com.azure:azure-ai-metricsadvisor-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-ai-textanalytics;5.1.0;5.2.0-beta.1
 com.azure:azure-ai-textanalytics-perf;1.0.0-beta.1;1.0.0-beta.1
+com.azure:azure-analytics-purview-account;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-analytics-purview-catalog;1.0.0-beta.1;1.0.0-beta.2
```
[the remark this answers is not in the document]
Comment (qwordy) [reply eng/versioning/version_client.txt:48]: Current version is 1.0.0-beta.2. It is not released.

**Pull request closed without merging**